### ISSUE TITLE
Low validation DSC when training on Brats2018

### ISSUE THREAD

**Issue by MohammedAdelFahmi** [black0017/MedicalZooPytorch#28]
Hello, First of all, thanks for the amazing work. 
Question: I tried to train the Unet3D model on bratS 2018 dataset using the 'train_brats_2018_new.py'. but I got lower validation DSC and higher validation loss, I think the model is facing some trouble on validation set: 
![image](https://user-images.githubusercontent.com/35504105/207441667-bc09464f-16a2-4093-8b7a-3a6df8a7a087.png)

**Comment by MohammedAdelFahmi** [comment]
@black0017

**Comment by yeshunlong** [comment]
Same problem, how to fix it?
In other words, what is the best args for every model and dataset?

**Comment by Gold3nk** [comment]
I have also encountered the same issue. When training on the brats2018 and brats2019 datasets, the dice scores of the validation set consistently converge to slightly above 60, which is significantly lower than the dice scores of the training set.
It seems that the inference code is incomplete as well. After attempting to fix it, I found that my dice score for the inference results is around 89, which seems incorrect because there shouldn't be such a large gap between the dice scores of my test set and training set, both of which come from the official training sets of brats2018 and brats2019. Due to my limited coding abilities, I had to abandon this code repository.

**Comment by WindymanJOX** [comment]
I think there is no proper DSC function for BratTS dataset in this repository. BratTS aims to seg WT, TC and ET, there are overlapping regions among those 3 areas, that is saying we should calculate the aboved 3 DSCs to show the performance of our trained model. But in this repo, they just put a general DSC func to all seg tasks.
